resources/views/livewire/ta-stats-monthly.blade.php

```diff
@@ -0,0 +1,76 @@
+<div>
+    <div class="row bg-warning w-100" wire:loading>
+        <h3 class="text-center"><i class="fa-solid fa-hourglass-start me-2"></i> Refreshing Dataset... please wait</h3>
+    </div>
+    <div class="row">
+        <div class="col-sm-2 col-xs-12">
+            <div class="card">
+                @php ($yearOfMonthsLookback = array())
+                @foreach(array_reverse(Carbon\CarbonPeriod::create(now()->subMonths(11), '1 month', now())->toArray()) as $date)
+                @php ($yearOfMonthsLookback[$date->format('m Y')] = $date->format('M Y'))
+                @endforeach
```

Comment:
Is this possible to be handled in the `app/Livewire` file instead of here and passed as a variable?